forge/test/mlir/test_ops.py

```diff
@@ -1909,3 +1909,85 @@ def forward(self, a, b):
     co_out = [co.to("cpu") for co in co_out]
     fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
     assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.xfail(
+    reason="RuntimeError: Found Unsupported operations while lowering from TTForge to TTIR in forward graph - repeat_interleave"
+)
+@pytest.mark.push
+def test_repeat():
+    class Repeat(nn.Module):
+        def __init__(self, repeats):
+            super().__init__()
+            self.repeats = repeats
+
+        def forward(self, x):
+            x = x + x
+            return x.repeat(*self.repeats)
+
+    inputs = [torch.rand(1, 2, 1, 4, 4)]
+
+    framework_model = Repeat(repeats=(1, 1, 4, 1, 1))
+    fw_out = framework_model(*inputs)
+
+    compiled_model = forge.compile(framework_model, sample_inputs=inputs)
+    co_out = compiled_model(*inputs)
+
+    co_out = [co.to("cpu") for co in co_out]
+    fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
+    assert all([compare_with_golden(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.xfail(
+    reason="RuntimeError: Found Unsupported operations while lowering from TTForge to TTIR in forward graph - repeat_interleave"
+)
+@pytest.mark.push
+def test_expand():
+    class Expand(nn.Module):
+        def __init__(self, expand_shape):
+            super().__init__()
+            self.expand_shape = expand_shape
+
+        def forward(self, x):
+            x = x + x
```

Comment:
Why do we need this one?

Reply:
Previously I was hit with `"There should only be one cpu pre function"` but in latest main I'm not getting this. Will remove this and update the PR

Reply:
Updated